spec/project_spec.rb

```diff
@@ -701,6 +716,21 @@ class InvalidErrorHandler; end
       expect(spy_logger).to have_received(:log).once.with(Logger::INFO, "Dispatching impression event to URL https://logx.optimizely.com/v1/events with params #{expected_params}.")
       expect(spy_logger).to have_received(:log).once.with(Logger::INFO, "Feature 'multi_variate_feature' is enabled for user 'test_user'.")
     end
+
+    it 'should return false, if the featureEnabled property is false' do
```

Comment:
Make it 
 it 'should return false, if the user is bucketed into a feature experiment but the featureEnabled property is false